<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="0" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11265" started-at="2018-04-18T02:52:55Z" finished-at="2018-04-18T02:53:06Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="11265" started-at="2018-04-18T02:52:55Z" finished-at="2018-04-18T02:53:06Z">
      <class name="test_Cases.CPNI_Tests">
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:test_Cases.CPNI_Tests@1c7cbad]" name="launchBrowser" is-config="true" duration-ms="11249" started-at="2018-04-17T20:52:55Z" finished-at="2018-04-17T20:53:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="FAIL" signature="tearDown()[pri:0, instance:test_Cases.CPNI_Tests@1c7cbad]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-17T20:53:06Z" finished-at="2018-04-17T20:53:06Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at test_Cases.CPNI_Tests.tearDown(CPNI_Tests.java:277)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:143)
	at org.testng.TestRunner.afterRun(TestRunner.java:980)
	at org.testng.TestRunner.run(TestRunner.java:611)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test_Cases.CPNI_Tests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
